calcom/cal.com
# Add an away mode to disable your booking page

# Away mode
<img width="755" alt="Screenshot 2022-01-05 at 15 38 39" src="https://user-images.githubusercontent.com/25907159/148245242-b421a325-78ca-4c31-a1c4-36fa2faddef3.png">

## What does this PR do?

Add a new boolean in the DB to set the user is away, add a toggle in the user dropdown to switch this value, and show a message + don't show event types on the booking page if the user's status is set to away.

## Type of change

## How should this be tested?

Clone, push DB schema, log in and set status to away via user dropdown and verify on the public page.
